feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/rrm_config.c

```diff
@@ -96,12 +112,35 @@ static bool rrm_config_changed( struct schema_Wifi_RRM_Config *old,
 	return false;
 }
 
+static bool rrm_radio_config_update(struct schema_Wifi_RRM_Config *conf )
+{
+	struct schema_Wifi_Radio_Config_flags changed;
+	struct schema_Wifi_Radio_Config rconf;
+
+	if (false == ovsdb_table_select_one(&table_Wifi_Radio_Config,
+			SCHEMA_COLUMN(Wifi_Radio_Config, freq_band), conf->freq_band, &rconf))
+	{
+		LOG(WARN, "Wifi_RRM_Config: No radio for band %s", conf->freq_band );
+		return false;
+	}
+
+	memset(&changed, 0, sizeof(changed));
+	target_radio_config_set2(&rconf, &changed);
+
+	return true;
+}
+
 static bool rrm_config_set( struct schema_Wifi_RRM_Config *old,
 	struct schema_Wifi_RRM_Config *conf ) 
 {
 	if (rrm_config_changed(old, conf)) {
-		return( rrm_config_update(conf, true));
+		rrm_config_update(conf, true);
```

Comment:
Lost the ability to report an error upstream.  Could create 'bool rc = true;' and "AND" it with the return from the two functions. 

I don't think it matters because we discard the return code a layer up.

Reply:
this is to avoid returning if both rrm_config_changed and rrm_radio_config_update conditions are true , we dont want to return from here without executing the rest of the function